src/components/play/brick/baseComponents/MathInHtml.tsx

```diff
@@ -18,22 +18,18 @@ class MathInHtml extends Component<MathHtmlProps> {
     }
 
     if (arr.length === 0) {
-      return <div>{this.props.value}</div>;
+      return <div >{this.props.value}</div>;
     }
 
     return (
-      <div>
-        {
```

Comment:
This makes me uncomfortable because it feels like a bracket is missing here somewhere. How would you feel about creating a Jest test for this https://jestjs.io/docs/en/tutorial-react

Reply:
Before this commit the section was missing a closing brace, but after this commit it seems fine as far as I can tell.